src/commands/image/imageSource/buildImageInAzure/ImageNameStep.ts

```diff
@@ -10,7 +10,7 @@ import { BuildImageInAzureImageSourceContext } from "./BuildImageInAzureContext"
 
 export class ImageNameStep extends AzureWizardPromptStep<BuildImageInAzureImageSourceContext> {
     public async prompt(context: BuildImageInAzureImageSourceContext): Promise<void> {
-        const suggestedImageName = await getSuggestedName(context, context.dockerfilePath);
+        const suggestedImageName = await getSuggestedName(context, context.rootFolder.uri.fsPath);
```

Comment:
I changed this because the dockerfile can be in the "src" folder, which causes the suggestion name to be "src". Figured that this is the behavior that we wanted.